assets/dev/js/editor/controls/media.js

```diff
@@ -24,15 +24,30 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		} );
 	},
 
+	/**
+	 * Get library type for `wp.media` using a given media type.
+	 *
+	 * @param {String} mediaType - The media type to get the library for.
+	 * @returns {String}
+	 */
+	getLibraryType: function( mediaType ) {
+		if ( mediaType ) {
+			return ( 'svg' === mediaType ) ? 'image/svg+xml' : mediaType;
+		}
+
+		return this.getMediaType();
```

Comment:
In case that the function is being called without an argument, the function returns this.getMediaType - but in this scenario the value that should be returned is "image/svg+xml" when the type is SVG.

Therefore, it should be something like:

```suggestion
		if ( ! mediaType ) {
			mediaType = this.getMediaType();
		}

		return ( 'svg' === mediaType ) ? 'image/svg+xml' : mediaType;
```